extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -471,6 +473,7 @@ export class ConsentUI {
     const iframe = this.parent_.ownerDocument.createElement('iframe');
     const sandbox = this.getSandboxAttribute_(promptUISrc);
     iframe.setAttribute('sandbox', sandbox);
+    iframe.setAttribute('allow', IFRAME_ALLOWED_PERMISSIONS.join('; '));
```

Comment:
Not doing it via config as commented here https://github.com/ampproject/amphtml/issues/39745#issuecomment-1956100455. Should I add any other permission?